<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addNewPerson</class>
 <widget class="QDialog" name="addNewPerson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_New_Name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>68</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Name: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_New_Gender">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>68</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_New_Byear">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Birth year: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_New_Dyear">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Death Year:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_New_name">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_New_Byear">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_new_Dyear">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_ADD_NewScientist">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Add scientist</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Cancel_NewScientist">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>320</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_male">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_female">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelErrorGender">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>-1 if still alive</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add new scientist</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
